red/runtime/storage/localfilesystem.js

```diff
@@ -390,8 +390,12 @@ var localfilesystem = {
                 headers += "// "+i+": "+meta[i]+"\n";
             }
         }
+        var data = headers+body;
```

Comment:
As this is used by different types of entry, such as the Function nodes, `body` isn't necessarily a flow that should be JSON stringified. It should only be done if `type==='flows'`. Also, `body` here will already be the string version of the flows - so you'll need to `JSON.parse()` it first before stringifying it to reformat it. The headers should be prepended _after_ it has been stringified.